.github/CODEOWNERS

```diff
@@ -202,10 +202,10 @@
 # ServiceOwners:                                                   @ryogok @TFR258 @tburns10 @areddish @toothache
 
 # PRLabel: %Cognitive - Face
-/sdk/cognitiveservices/Vision.Face/                                @JinyuID @dipidoo @SteveMSFT @msyache @longli0 @ShaoAnLin @lulululululu
+/sdk/face/                                                         @JinyuID @dipidoo @longli0 @ShaoAnLin @lulululululu @leareai @Han-msft
```

Comment:
Do you know the existing set of contacts in here?   All of them are in an invalid state and need to fix their memberships/permissions.

Reply:
Yes I know them, let me send a mail to notify them.